exporter/signalfxexporter/translation/delta_translator.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package translation
+
+import (
+	"github.com/gogo/protobuf/proto"
+	sfxpb "github.com/signalfx/com_signalfx_metrics_protobuf/model"
+)
+
+type deltaTranslator struct {
+	prevPts map[string]*sfxpb.DataPoint
+}
+
+func newDeltaTranslator() *deltaTranslator {
+	return &deltaTranslator{prevPts: map[string]*sfxpb.DataPoint{}}
+}
+
+func (t *deltaTranslator) translate(processedDataPoints []*sfxpb.DataPoint, tr Rule) []*sfxpb.DataPoint {
```

Comment:
I wouldn't expect a translate function to have side effects.  Maybe `appendDeltaPts()`?

Reply:
my mistake: `processedDataPoints` is unchanged, but the name is a bit unclear imo

Reply:
No worries. processedDataPoints is the name of the variable at the call site. I've changed it here to just `pts`.